src/server/services/collection.service.ts

```diff
@@ -603,14 +606,46 @@ export const upsertCollection = async ({
         where: { collectionId: updated.id },
         data: { publishedAt: new Date() },
       });
-    } else {
+    } else if (updated.mode !== CollectionMode.Contest) {
```

Comment:
I think we shouldn't allow this for `Bookmark` collections either
```suggestion
    } else if (!updated.mode) {
```